src/metabase/config.clj

```diff
@@ -135,18 +143,15 @@
   mb-version-and-process-identifier
   (format "%s [%s]" mb-app-id-string local-process-uuid))
 
-(defn- mb-session-cookie-samesite*
-  []
+(defn mb-session-cookie-samesite
+  "Value for session cookie's `SameSite` directive. Must be one of \"none\", \"lax\", or \"strict\" (case insensitive)."
+  ^Keyword []
```

Comment:
this `^Keyword` typehint strikes me as quite strange. Are we using interop on the keyword class anywhere?

Reply:
I kept it just because it was there before (see `(def ^Keyword mb-session-cookie-samesite ,,,)` below). Looking at references to this functions (previously: var) I do not see Java interop, so I will remove it.

Reply:
https://github.com/metabase/metabase/pull/31716/commits/523d91a25d074824e8a40f7ef46f74ec30018b37